drivers/bridge/bridge.go

```diff
@@ -570,6 +570,10 @@ func (d *driver) getNetworks() []*bridgeNetwork {
 func (d *driver) CreateNetwork(id types.UUID, option map[string]interface{}) error {
 	var err error
 
+	if d.config == nil {
+		return fmt.Errorf("bridge driver configuration is missing")
```

Comment:
Little tip, for static errors like this its nice to define an exported error type like 

`var ErrNoBridgeConfiguration = errors.New("bridge driver configuration is missing")`

This allows client code to check the error in a type safe way instead of relying on the error strings.